adapters/repos/db/vector/hnsw/commitlog/logger.go

```diff
@@ -107,6 +109,44 @@ func (l *Logger) AddSQCompression(data compressionhelpers.SQData) error {
 	return err
 }
 
+func (l *Logger) AddMuvera(data multivector.MuveraData) error {
```

Comment:
Need to add a test like `TestCondensorWithPQInformation` for Muvera information. This ensures when condensing commit logs everything it retains.